content/docs/started/getting-started-gke.md

```diff
@@ -80,12 +80,12 @@ Run the following steps to deploy Kubeflow:
 1. Run the following script to download `kfctl.sh`:
 
     ```
-    mkdir ${KUBEFLOW_SRC}
-    cd ${KUBEFLOW_SRC}
+    mkdir ${KUBEFLOW_REPO}
+    cd ${KUBEFLOW_REPO}
     export KUBEFLOW_TAG={{% kf-stable-tag %}}
     curl https://raw.githubusercontent.com/kubeflow/kubeflow/{{< params "githubbranch" >}}/scripts/download.sh | bash
```

Comment:
Do we need to update the download script as well to look for the appropriate env var?

Reply:
i did't find this env KUBEFLOW_SRC in the download.sh.  In the doc, i think KUBEFLOW_SRC is same as KUBEFLOW_REPO